zproject/backends.py

```diff
@@ -63,7 +65,7 @@
     EmailContainsPlusError, PreregistrationUser, UserProfile, Realm, custom_profile_fields_for_realm, \
     get_user_profile_by_id, remote_user_to_email, \
     email_to_username, get_realm, get_user_by_delivery_email, supported_auth_backends
-
+from zproject.settings import SOCIAL_AUTH_FIELDS_STORED_IN_SESSION
```

Comment:
I think we should avoid this import and just use `settings.SOCIAL_...` the way we always do :thinking:

Reply:
changed.